### ISSUE TITLE
score object needs resize manually

### ISSUE THREAD

**Issue by jlquispec** [cac-t-u-s/om-sharp-users#41]
my preferences:
![image](https://user-images.githubusercontent.com/53123770/127380795-673f9bd2-cc77-409e-8b48-cf48d144cadd.png)

when I create a new score object it's necessary to resize the object manually:
![image](https://user-images.githubusercontent.com/53123770/127380892-f6fb80d8-8163-4a72-a3f3-3dd9e06562fe.png)

![image](https://user-images.githubusercontent.com/53123770/127380932-ce2f4a2e-5230-4fdb-bff8-af3511611f0f.png)

**Comment by j-bresson** [comment]
The "Score font size" preference determines the "default" font size, but not the default size of the box (there is no preference for that :).
.. and as soon as the default font size doesn't let the score object fit in the box's height, it is scaled down.
So, I would say this is the expected behavior... but you would maybe like to have a preference for the default size as well ..

**Comment by jlquispec** [comment]
Yes. I think to have a preference for default size of box could help a lot because normally the action to resize the initial size box when an score object is created is neccesary to continue editing 

![image](https://user-images.githubusercontent.com/53123770/130395422-cd272370-a220-4d0a-9480-dee084b5fdb2.png)
